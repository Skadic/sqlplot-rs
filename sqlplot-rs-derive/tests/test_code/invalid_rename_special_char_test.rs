use sqlplot_rs_derive::ResultLine;

#[derive(ResultLine)]
struct MyStruct {
    #[result(name="6atoken")]
    a: &'static str,
    b: u32,
    c: f64,
}
fn main() {}