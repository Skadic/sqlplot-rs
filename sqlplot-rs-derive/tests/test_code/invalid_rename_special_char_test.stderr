error: must start with an alphabetic character or underscore
 --> tests/test_code/invalid_rename_special_char_test.rs:5:19
  |
5 |     #[result(name="6atoken")]
  |                   ^^^^^^^^^
